### ISSUE TITLE
NVDA+Alt+D failing to report status or move to Dropbox tray icon

### ISSUE THREAD

**Issue by ScottChesworth** [nvdaaddons/dropbox#1]
Hi,

I'm running the Dropbox add-on V4.7, NVDA 2022.1, and Dropbox 152.4.4880 here on Windows 10 with all current updates. All of a sudden, NVDA+Alt+D has stopped working. Here's what I've tried so far:
1. Restarted NVDA.
2. Restarted Dropbox.
3. Restarted Windows.
4. Reinstalled the add-on, making sure that %appdata%\NVDA\Addons was clean, tried both V4.6 and 4.7.
5. Reinstalled Dropbox app.
6. Checked in NVDA Prefs -> Input Gestures that the shortcut is assigned for all keyboard layouts as expected.
7. Thought long and hard about what I might have missed, but that didn't seem to help at all :)

If you have any other ideas, I'd appreciate the help!

Thanks in advance.
